### DISCUSSION TITLE
blog/happy-new-year-2024/

### DISCUSSION THREAD

**Discussion by giscus** [sqybi/SQYBI.com#13] [Announcements]
# blog/happy-new-year-2024/

虽然一下子就是大半个月没有更新，但在这个时间还是让我先说一句，新年快乐！

https://sqybi.com/blog/happy-new-year-2024/

**Comment by Oreoxmt** [comment]
giscus 终于好了！

**Reply by sqybi** [reply]
是的（其实好了一段时间了）
之前是 repo 权限设置错了……
